quint/src/builtin.ts

```diff
@@ -0,0 +1,34 @@
+/* ----------------------------------------------------------------------------------
+ * Copyright (c) Informal Systems 2023. All rights reserved.
+ * Licensed under the Apache 2.0.
+ * See License.txt in the project root for license information.
+ * --------------------------------------------------------------------------------- */
+
+/**
+ * Builtin operators and values, defined in `builtin.qnt`
+ *
+ * @author Gabriela Moreira
+ *
+ * @module
+ */
+
+import { DocumentationEntry, produceDocs } from "./docs";
+import { resolve } from 'path'
+import { readFileSync } from "fs";
+import { ErrorMessage, parsePhase1 } from "./quintParserFrontend";
+import { Either } from "@sweet-monads/either";
+import { lf } from "eol";
+
+/**
+ * The documentation map for the builtin definitions
+ *
+ * @returns a map of builtin definition names to their documentation
+ */
+export function builtInDocs(): Either<ErrorMessage[], Map<string, DocumentationEntry>> {
```

Comment:
it looks like you treat this as one word everywhere else:

```suggestion
export function builtinDocs(): Either<ErrorMessage[], Map<string, DocumentationEntry>> {
```